packages/tailwind-preset/src/tailwind.preset.ts

```diff
@@ -285,10 +267,88 @@ const mantlePreset = {
 				unchecked: 'checked="false"',
 			},
 			backgroundColor: {
-				body: "hsl(var(--body) / <alpha-value>)",
-				background: "hsl(var(--background) / <alpha-value>)",
-				popover: "hsl(var(--popover) / <alpha-value>)",
-				card: "hsl(var(--card) / <alpha-value>)",
+				base: "hsl(var(--bg-base))",
+				card: "hsl(var(--bg-card))",
+				popover: "hsl(var(--bg-popover))",
+				form: "hsl(var(--bg-form))",
+				neutral: "hsl(var(--bg-neutral))",
+				primary: "hsl(var(--bg-primary))",
+				danger: "hsl(var(--bg-danger))",
+				warning: "hsl(var(--bg-warning))",
+				success: "hsl(var(--bg-success))",
+				"base-hover": "hsl(var(--bg-base-hover))",
+				"card-hover": "hsl(var(--bg-card-hover))",
+				"popover-hover": "hsl(var(--bg-popover-hover))",
+				"form-hover": "hsl(var(--bg-form-hover))",
+				"neutral-hover": "hsl(var(--bg-neutral-hover))",
+				"primary-hover": "hsl(var(--bg-primary-hover))",
+				"danger-hover": "hsl(var(--bg-danger-hover))",
+				"warning-hover": "hsl(var(--bg-warning-hover))",
+				"success-hover": "hsl(var(--bg-success-hover))",
+				"form-active": "hsl(var(--bg-form-active))",
+				"neutral-active": "hsl(var(--bg-neutral-active))",
+				"primary-active": "hsl(var(--bg-primary-active))",
+				"danger-active": "hsl(var(--bg-danger-active))",
+				"warning-active": "hsl(var(--bg-warning-active))",
+				"success-active": "hsl(var(--bg-success-active))",
+				"neutral-muted": "hsl(var(--bg-neutral-muted))",
+				"primary-muted": "hsl(var(--bg-primary-muted))",
+				"danger-muted": "hsl(var(--bg-danger-muted))",
+				"warning-muted": "hsl(var(--bg-warning-muted))",
+				"success-muted": "hsl(var(--bg-success-muted))",
+				"neutral-muted-hover": "hsl(var(--bg-neutral-muted-hover))",
+				"primary-muted-hover": "hsl(var(--bg-primary-muted-hover))",
+				"danger-muted-hover": "hsl(var(--bg-danger-muted-hover))",
+				"warning-muted-hover": "hsl(var(--bg-warning-muted-hover))",
+				"success-muted-hover": "hsl(var(--bg-success-muted-hover))",
+				"neutral-muted-active": "hsl(var(--bg-neutral-muted-active))",
+				"primary-muted-active": "hsl(var(--bg-primary-muted-active))",
+				"danger-muted-active": "hsl(var(--bg-danger-muted-active))",
+				"warning-muted-active": "hsl(var(--bg-warning-muted-active))",
+				"success-muted-active": "hsl(var(--bg-success-muted-active))",
+			},
+			textColor: {
+				neutral: "hsl(var(--text-neutral))",
+				primary: "hsl(var(--text-primary))",
```

Comment:
Noting that we'll need to audit use of `text-primary` in production, since I think those exist and aren't a blue. We should also include `text-secondary`. I think that's what you're hinting at with `text-neutral`.

Reply:
yeah, we will need to be very careful if we keep this when we integrate back into production

Reply:
Makes sense. and Yeah, agreed. I actually haven't done any main text tokens yet. Those are what I want to dive into next. `text-neutral` just mirrors the other functional text colors, but you're right we may not actually use it.